tldr.py

```diff
@@ -338,8 +346,12 @@ def output(page):
 
 def update_cache(language=None):
     if language is None:
-        default_lang = get_default_language()
-        language = default_lang if default_lang is not None else 'en'
+        tldr_language = os.environ.get("TLDR_LANGUAGE")
+        if tldr_language is not None:
+            language = tldr_language
+        else:
+            default_lang = get_default_language()
+            language = default_lang if default_lang is not None else 'en'
```

Comment:
I believe we can shorten this to be just two lines?
```suggestion
        tldr_language = os.environ.get("TLDR_LANGUAGE", get_default_language())
        language = default_lang if default_lang is not None else 'en'
```

Reply:
You could shorten a tad bit further by saying `default_lang if default_lang else 'en'` (i.e. removing ` is not None`).

Reply:
I added this suggestion, except of course that it should be `tldr_language` instead of `default_lang` in the last line